### PR TITLE
Modifications to inference service for control plane logs

### PR DESCRIPTION
**Pull request by AmartC**
This PR includes a couple modifications for the control plane service. The inference service is reading from a new Nats subject and rather than using the script field for updating Opensearch, the anomaly level field is directly being passed in as a field when updating logs back into Opensearch.

### CODE REVIEW

File: models/nulog/masker.py [hunk 1]
```diff
@@ -28,12 +28,27 @@ def __init__(
...
+            modified_content = content[:]
+
+            for m in matching_indices:
+                substring = content[m[0] : m[1]]
+                left_side_term = substring.split('="')[0]
+                if left_side_term != "err":
+                    modified_content = modified_content.replace(
+                        substring, f"{left_side_term}=<{left_side_term}>"
+                    )
+            content = modified_content[:]
```
Comment (sanjay920) [inline models/nulog/masker.py:51]: same as https://github.com/rancher/opni-preprocessing-service/pull/19#discussion_r839085638

File: nulog-inference-service/start-nulog-inference.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ async def consume_logs(logs_queue):
     """
     if IS_CONTROL_PLANE_SERVICE:
         await nw.subscribe(
-            nats_subject="preprocessed_logs_control_plane",
+            nats_subject="nulog_cp_logs",
```
Comment (sanjay920) [inline nulog-inference-service/start-nulog-inference.py:96]: there should be no more reference to `nulog` moving forward

**Review by sanjay920 — DISMISSED** [review]

File: models/nulog/masker.py [hunk 3]
```diff
@@ -28,12 +28,27 @@ def __init__(
...
-    def mask(self, content: str):
+    def mask(self, content: str, is_control_plane_log: bool):
 
         # get rid of escape keys
         content = self.ansi_escape.sub("", content)
 
-        # reduce any json objects (TODO)
+        # If log is a control plane log, mask out content from structured control plane log messages which follow the 'a="somecontent" to be a=<a>'
+        if is_control_plane_log:
+            assignment_regex = '([A-z]+)="(.*?)"'
```
Comment (kralicky) [inline models/nulog/masker.py:38]: `A-z` is improper regex - the character sets A-Z and a-z are not contiguous in the ascii table. If you want to match uppercase and lowercase letters you need `a-zA-Z`

Comment (AmartC) [reply models/nulog/masker.py:38]: Hmm, so I tried this regex out again just now to be sure and it is working intended for upper or lower case patterns.

Comment (kralicky) [reply models/nulog/masker.py:38]: It will match upper and lower case letters. It will also match everything between Z and a.

Comment (AmartC) [reply models/nulog/masker.py:38]: Yeah, but would those appear in that format specifically within control plane log messages?

File: nulog-inference-service/start-nulog-inference.py (hunk 2 above)
Comment (AmartC) [reply nulog-inference-service/start-nulog-inference.py:96]: Okay, I'll change all the nulog references moving forward to general Deep Learning model references.

File: models/nulog/masker.py (hunk 3 above)
Comment (AmartC) [reply models/nulog/masker.py:38]: Let me look into this some more.

**Review by sanjay920 — APPROVED** [review]

**Pull request merged**